src/storage/mutate/AddVerticesProcessor.cpp

```diff
@@ -181,6 +177,9 @@ AddVerticesProcessor::addVertices(PartitionID partId,
                         if (env_->checkRebuilding(spaceId_, partId, indexId)) {
                             auto deleteOpKey = OperationKeyUtils::deleteOperationKey(partId);
                             batchHolder->put(std::move(deleteOpKey), std::move(oi));
+                        } else if (env_->checkIndexLocked(spaceId_, partId, indexId)) {
+                            LOG(ERROR) << "The part have locked";
+                            return folly::none;
```

Comment:
I think move line 180->182 and 207-209 to 166 is better. what do you think?

Reply:
Maybe similar with the next point. why don't merge the status check into one？Because I could not make sure the status would not change. If we check the status at before, it maybe pass the check but still could change to locked. So I prefer check it when need save the modify.